packages/discovery-provider/integration_tests/tasks/test_index_payment_router.py

```diff
@@ -258,8 +257,8 @@ def test_process_payment_router_tx_details_transfer_without_purchase(
         assert transaction_record.transaction_type == USDCTransactionType.transfer
         assert transaction_record.method == USDCTransactionMethod.receive
         assert transaction_record.change == 1000000
-        # For transfers, the metadata is the source address
-        assert transaction_record.tx_metadata == transactionSenderUsdcAccount
+        # For transfers, source is the owning wallet unless it's a transfer from a user bank
+        assert transaction_record.tx_metadata == transactionSenderAddress
```

Comment:
minor nit: could make it even more clear in var name with `transactionSenderOwnerAccount` or `transactionSenderRootAccount` or something. but nbd the comment is pretty clear.

Reply:
Heard! Feel like I could go around a hundred times trying to figure out the clearest way to describe this and still come out thinking it's confusing :-D

Reply:
yeah same

Reply:
Updated